### PR TITLE
ReleaseNotes update

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Merged lost release notes from v2.8.1;
- Added v2.9.0 template;

### Reason for changes

 - Upcoming release;

### Related tickets

- 132634

#### For the contributors: 

1. Please add your changes (as the commit to the branch) to the list according to the template and previous notes;
2. Do not add tests-related notes;
3. Provide the list of the PRs (for all your notes) in the comment for the discussion;

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (ONNX) Introduced support for the ONNX backend in the `nncf.quantize_with_accuracy_control()` method. Users can now perform quantization with accuracy control for `onnx.ModelProto`. By leveraging this feature, users can enhance the accuracy of quantized models while minimizing performance impact.
+  - (ONNX) Added an example based on the YOLOv8n-seg model for demonstrating the usage of quantization with accuracy control for the ONNX backend.
```
Comment (andrey-churkin) [inline ReleaseNotes.md:13]: PRs: #2399, #2465

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
 
+## New in Release 2.9.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
```
Comment (andreyanufr) [inline ReleaseNotes.md:11]:
```suggestion
- Features:
  - (OpenVINO) Added modified AWQ algorithm for 4-bit data-aware weights compression. This algorithm applied only for patterns `MatMul->Multiply->Matmul`. For that `awq` optional parameter has been added to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this option increases the compression time).
```

**Review by andreyanufr — APPROVED** [review]

**Comment by MaximProshin** [comment]
@l-bat , @nikita-savelyevv , please add the list of new/updated with NNCF notebooks

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
+  - [Post-Training Optimization of Stable Diffusion v2 Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image.ipynb)
+  - [Post-Training Optimization of DeciDiffusion Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/259-decidiffusion-image-generation/259-decidiffusion-image-generation.ipynb)
+  - [Post-Training Optimization of DepthAnything Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/280-depth-anything/280-depth-anything.ipynb)
+  - [Post-Training Optimization of Stable Diffusion ControlNet Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/235-controlnet-stable-diffusion/235-controlnet-stable-diffusion.ipynb)
+
+Compression-aware training:
+
+- Fixes
+  - (PyTorch) Fixed issue with `NNCFNetworkInterface.get_clean_shallow_copy` missed arguments.
+
```
Comment (alexsu52) [inline ReleaseNotes.md:39]: Are you going to add "Thanks to our contributors" in this release https://github.com/openvinotoolkit/nncf/issues/2499?

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by andrey-churkin — APPROVED** [review]

File: ReleaseNotes.md (hunk 3 above)
Comment (KodiaqQ) [reply ReleaseNotes.md:39]: Do we need it in `ReleaseNotes.md`? Which format do you propose to use in the document?

Comment (alexsu52) [reply ReleaseNotes.md:39]: I think the release notes is good place to say thanks to our contributors. Format is following:
### Thanks to our Contributors
contributor_name_0, contributor_name_1 and etc.

Comment (KodiaqQ) [reply ReleaseNotes.md:39]: I'll place the information about contributors in the GitHub notes during the release. Do we need it in the `ReleaseNotes.md` **document**?

Comment (alexsu52) [reply ReleaseNotes.md:39]: Good suggestion! It's enough.

File: docs/Installation.md [hunk 4]
```diff
@@ -66,6 +66,7 @@ as well as the supported versions of Python:
 | NNCF      | OpenVINO   | PyTorch  | ONNX     | TensorFlow | Python |
 |-----------|------------|----------|----------|------------|--------|
 | `develop` | `2023.3.0` | `2.2.1`  | `1.13.1` | `2.12.0`   | `3.8`  |
```
Comment (MaximProshin) [inline docs/Installation.md:68]: Shouldn't 2.9.0 be added here?

File: ReleaseNotes.md [hunk 5]
```diff
@@ -1,5 +1,53 @@
...
+## New in Release 2.9.0
+
+Post-training Quantization:
+
+- Features:
+  - (OpenVINO) Added modified AWQ algorithm for 4-bit data-aware weights compression. This algorithm applied only for patterns `MatMul->Multiply->Matmul`. For that `awq` optional parameter has been added to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this option increases the compression time).
+  - (ONNX) Introduced support for the ONNX backend in the `nncf.quantize_with_accuracy_control()` method. Users can now perform quantization with accuracy control for `onnx.ModelProto`. By leveraging this feature, users can enhance the accuracy of quantized models while minimizing performance impact.
+  - (ONNX) Added an example based on the YOLOv8n-seg model for demonstrating the usage of quantization with accuracy control for the ONNX backend.
+  - (PT) Added SmoothQuant algorithm for PyTorch backend in `nncf.quantize()`.
+  - (OpenVINO) Added an example with the hyperparameters tuning for the TinyLLama model.
```
Comment (MaximProshin) [inline ReleaseNotes.md:12]: Please add the reference

File: ReleaseNotes.md [hunk 6]
```diff
@@ -1,5 +1,53 @@
...
+- Improvements:
+  - (ONNX) Aligned statistics collection with OpenVINO and PyTorch backends.
+  - Extended `nncf.compress_weights()` with Convolution & Embeddings compression in order to reduce memory footprint.
+- Deprecations/Removals:
+  - (OpenVINO) Removed outdated examples with `nncf.quantize()` for BERT and YOLOv5 models.
+  - (OpenVINO) Removed outdated example with `nncf.quantize_with_accuracy_control()` for SSD MobileNetV1 FPN model.
+  - (PyTorch) Deprecated the `binarization` algorithm.
+  - Removed Post-training Optimization Tool as OpenVINO backend.
+  - Removed Dockerfiles.
+  - Removed the `TargetDevice.VPU` device.
```
Comment (MaximProshin) [inline ReleaseNotes.md:28]: Add that it was replaced by NPU

File: docs/Installation.md (hunk 4 above)
Comment (KodiaqQ) [reply docs/Installation.md:68]: See - https://github.com/openvinotoolkit/nncf/pull/2545

File: ReleaseNotes.md (hunk 6 above)
Comment (KodiaqQ) [reply ReleaseNotes.md:28]: See the line 15 above.

File: ReleaseNotes.md (hunk 5 above)
Comment (KodiaqQ) [reply ReleaseNotes.md]: Done

File: docs/Installation.md (hunk 4 above)
Comment (MaximProshin) [reply docs/Installation.md:68]: ok

File: ReleaseNotes.md (hunk 6 above)
Comment (MaximProshin) [reply ReleaseNotes.md:28]: I saw it there, but let it be specific that it's not just removal if one reads just this section.

Comment (KodiaqQ) [reply ReleaseNotes.md]: Updated.

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**